### PR TITLE
refactor download only test for better visibility

### PR DESCRIPTION
**Pull request by sharifelgamal**
TestDownloadOnly had one subtest ("check_json_event') but still had other tests running in the parent, which makes it difficult for gopogh to parse failures in the parent. This PR changes the tests so that we only have subtests, so we know exactly what fails.

### CODE REVIEW

**Comment by sharifelgamal** [comment]
The tests will get fixed once #10196 is merged

File: test/integration/aaa_download_only_test.go [hunk 1]
```diff
@@ -54,26 +53,31 @@ func TestDownloadOnly(t *testing.T) {
...
-
-			rt, err := Run(t, exec.CommandContext(ctx, Target(), args...))
-			if rrr == nil {
-				// Preserve the initial run-result for debugging
-				rrr = rt
-			}
-			if err != nil {
-				t.Errorf("failed to download only. args: %q %v", args, err)
-			}
 			t.Run("check json events", func(t *testing.T) {
```
Comment (medyagh) [inline test/integration/aaa_download_only_test.go:65]: how about choosing shorter names ? so it shows better in gopogh url ?
"json-events" instead "check json events"

Comment (medyagh) [reply test/integration/aaa_download_only_test.go:65]: and same for other names

**Review by medyagh — CHANGES_REQUESTED** [review]

**Review by medyagh — APPROVED** [review]

File: test/integration/aaa_download_only_test.go (hunk 1 above)
Comment (sharifelgamal) [reply test/integration/aaa_download_only_test.go]: done

**Pull request merged**